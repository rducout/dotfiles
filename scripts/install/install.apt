meld
xclip
build-essential
git
google-chrome-stable
nodejs
npm
zsh