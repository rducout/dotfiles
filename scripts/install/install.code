akamud.vscode-theme-onedark
dbaeumer.vscode-eslint
Equinusocio.vsc-material-theme
GrapeCity.gc-excelviewer
jasonnutter.search-node-modules
ms-python.python
ms-vscode.csharp
msjsdiag.debugger-for-chrome
Orta.vscode-jest
PKief.material-icon-theme
rokoroku.vscode-theme-darcula
trinm1709.dracula-theme-from-intellij
Unity.unity-debug