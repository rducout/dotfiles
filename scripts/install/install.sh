#!/bin/bash
# Install everything I need
# Romain Ducout

DIRNAME=$(dirname "$0")

( exec $DIRNAME/install_packages.sh )
( exec $DIRNAME/install_code_packages.sh )
