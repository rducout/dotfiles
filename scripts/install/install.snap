code
intellij-idea-community
slack
inkscape
gimp
vlc
skype